src/corehost/cli/deps_resolver.cpp

```diff
@@ -486,6 +486,16 @@ void deps_resolver_t::resolve_probe_dirs(
     {
         if (probe_entry_in_configs(entry, &candidate))
         {
+            // For standalone apps, on win7, coreclr needs ApiSets which has to be in the DLL search path.
+            if (!m_portable && pal::need_api_sets()
+                    && ends_with(entry.library_name, _X("Microsoft.NETCore.Windows.ApiSets"), false))
+            {
+                const pal::string_t api_sets_dir = action(candidate);
+                if (api_sets_dir != clr_dir)
+                {
+                    m_api_set_paths.insert(api_sets_dir);
```

Comment:
So long as we will have only a single path in the set, I am fine with the change.